src/nodes/ImageView.vue

```diff
@@ -224,16 +226,13 @@ export default {
 
 			return this.loaded && this.imageLoaded
 		},
-		imageFileId() {
-			return getQueryVariable(this.src, 'fileId')
-		},
 		isSupportedImage() {
-			return typeof this.mime === 'undefined'
-				|| IMAGE_MIMES.indexOf(this.mime) !== -1
+			return typeof this.mimeType === 'undefined'
+				|| [...IMAGE_MIMES, ...MEDIA_MIMES].indexOf(this.mimeType) !== -1
 		},
 		internalLinkOrImage() {
-			if (this.imageFileId) {
-				return generateUrl('/f/' + this.imageFileId)
+			if (this.fileId) {
+				return generateUrl('/apps/files/f/' + this.fileId)
```

Comment:
Same here, no need for the change. The URL `/f/<fileId>` works just fine, no need for '/apps/files/f/<fileId>`, no?

Reply:
@mejo- `/f/<fileId>` will not work, I have checked this.